src/components/SystemMessageNotification/SystemMessageNotification.tsx

```diff
@@ -0,0 +1,28 @@
+import React, {useState} from 'react';
+import {Alert} from 'react-bootstrap';
+import styles from './SystemMessageNotification.module.scss';
+
+
+
+export type SystemMessageNotificationTypes = 'info' | 'warning' | 'notice';
+
+interface Props {
+  onClick?: () => void;
+  type: SystemMessageNotificationTypes;
+  description: string;
+}
+
+export function SystemMessageNotification (props: Props) {
+
+    const [show] = useState(true);
+
+        return show ? (
+          <div className={styles['alert-container']}>
+             <Alert className={styles[props.type]} dismissible={true}  onClose={props.onClick}>
```

Comment:
Your problem is here. Your API endpoint is serving the message type with a capitalized first letter, but your classnames in your CSS file are all lowercase.

You can fix this in two ways: either change the CSS file and `SystemMessageNotificationTypes` to use capital first letters, or change this line to use `className={styles[props.type.toLowerCase()]}`.

I don't know if @jasekiw has a preference but I think the former method is cleaner.